src/media/js/main.js

```diff
@@ -151,7 +151,9 @@ function(_) {
         // are in a packaged app, wait for the iframe to be loaded, otherwise
         // we are using the direct installer and we just need to wait for the
         // normal loaded event.
-        var event_for_apps = window.location.protocol === 'app:' ? 'iframe-install-loaded' : 'loaded';
+        var is_packaged_app = window.location.protocol === 'app:';
+        var is_iframed_app = window.top != window.self;  // Crude, but enough for our needs.
+        var event_for_apps = is_packaged_app ? 'iframe-install-loaded' : 'loaded';
```

Comment:
I could be misreading this, but this looks backwards: emit an event named `iframe-install-loaded` for the non-iframed packaged app?